src/components/FoodOrderComponents/OrderPlace.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+    <v-container class="orderPlaceStyle">
+        <!-- Header & Input -->
+        <v-row justify="center" no-gutters>
+            <h2>מאיפה תרצו להזמין?</h2>
+        </v-row>
+
+        <v-row justify="center" no-gutters>
+            <v-col class="placeTextFieldStyle" cols="3">
+                <v-text-field
+                        :append-outer-icon="placeInput ? 'navigate_before' : ''"
+                        @click:append-outer="sendPlaceInput"
+                        @click:clear="clearPlaceInput"
+                        clear-icon="mdi-close-circle"
+                        clearable
+                        filled
+                        label="מקום"
+                        type="text"
+                        v-model="placeInput"
+                ></v-text-field>
+            </v-col>
+        </v-row>
+
+        <!-- Places list -->
+        <v-row justify="center" no-gutters>
+            <v-col cols="3" class="placeListsCol">
+                <v-progress-circular :size="40" color="primary"
+                                     indeterminate v-if="$apollo.loading"/>
```

Comment:
Use global spinner

Reply:
There is a problem locally with that. For now I keep it that way untill will get an answer.